test/activities/quizzes/data/attempts/QuizAttemptsEntity.js

```diff
@@ -0,0 +1,239 @@
+export const quizAttemptsEntity = {

```

Comment:
Should the sub-entities be extacted into other files?

Reply:
Probably fine as is. Unless there are more variations of sub-entities you would like to test.